src/app/services/game-play.service.ts

```diff
@@ -0,0 +1,99 @@
+import { Injectable } from '@angular/core';
+import { BehaviorSubject, fromEvent } from 'rxjs';
+import { map } from 'rxjs/operators';
+
+import { AnswerService } from './answer.service';
+import { Answer } from '../models/answer';
+
+interface Game {
+  answer: Answer;
+  acceptedGuesses: string[];
+  status: Status;
+  currentGuessIdx: number;
+  currentGuess: string;
+};
+
+enum Status {
+  playing,
+  won,
+  lost
+};
+
+const emptyGame: Game = {
+  answer: <Answer>{},
+  acceptedGuesses: [],
+  status: Status.playing,
+  currentGuessIdx: 0,
+  currentGuess: ''
+};
+
+@Injectable({
+  providedIn: 'root'
+})
+export class GamePlayService {
+  private readonly game$ = new BehaviorSubject<Game>(emptyGame);
```

Comment:
I like BehaviorSubjects because `getValue()` is so handy. Maybe you already read this, but a Subject doesn't emit its most recent value when you subscribe whereas BehaviorSubjects and ReplaySubjects do.
I'm not sure if the game needs to be a subject/observable at all but I'm glad you're using one for it and BehaviorSubject makes sense to me.